ui/src/components/dialog/__tests__/QDialog.cy.js

```diff
@@ -26,103 +136,425 @@ describe('Dialog API', () => {
+          mountQDialogWrapper({
+            props: {
+              ...vModelAdapter(model),
+              noFocus: false
+            }
+          })
+
+          cy.withinDialog(() => {
+            cy.focused()
+              .should('have.class', 'q-dialog__inner')
+            closeDialogViaBackdrop()
+          })
+
+          cy.wrap().then(async () => {
+            await Cypress.vueWrapper.setProps({ noFocus: true })
+            model.value = true
+            await nextTick()
+          })
+
+          cy.withinDialog(() => {
+            cy.focused().should('not.exist')
+            closeDialogViaBackdrop()
+          })
         })
       })
 
       describe('(prop): no-shake', () => {
-        it.skip(' ', () => {
-          //
+        it('should not shake dialog', () => {
+          const model = ref(true)
+          mountQDialogWrapper({
+            props: {
+              ...vModelAdapter(model),
+              persistent: true
+            }
+          })
+
+          cy.withinDialog({
+            persistent: true,
+            fn: () => {
+              closeDialogViaBackdrop()
+              cy.get('.q-dialog__inner')
+                .should('have.class', 'q-animate--scale')
+            }
+          })
+
+          assertPersistentDialogExists()
+
+          cy.wrap().then(async () => {
+            await Cypress.vueWrapper.setProps({ noShake: true })
+          })
+
+          cy.withinDialog({
+            persistent: true, fn: () => {
+              closeDialogViaBackdrop()
+              cy.get('.q-dialog__inner')
+                .should('not.have.class', 'q-animate--scale')
+            }
+          })
+
+          assertPersistentDialogExists()
         })
       })
     })
 
     describe('Category: content', () => {
       describe('(prop): seamless', () => {
-        it.skip(' ', () => {
-          //
+        it('should put the dialog in a seamless state', () => {
+          const model = ref(true)
+          mountQDialogWrapper({
+            props: {
+              ...vModelAdapter(model),
+              seamless: true
+            }
+          })
+
+          cy.dataCy('dialog-form')
+            .should('have.class', 'q-dialog--seamless')
+
+          cy.dataCy('input-field')
+            .should('be.visible')
+            .then(async () => {
+              await Cypress.vueWrapper.setProps({ seamless: false })
+            })
+
+          cy.dataCy('dialog-form')
+            .should('not.have.class', 'q-dialog--seamless')
+          cy.dataCy('input-field')
+            .should('not.be.visible')
         })
       })
 
       describe('(prop): maximized', () => {
-        it.skip(' ', () => {
-          //
+        it('should maximize the dialog', () => {
+          const model = ref(false)
+          mountQDialogWrapper({
+            props: {
+              ...vModelAdapter(model),
+              maximized: true
+            }
+          })
+
+          cy.dataCy('dialog-page').then(() => {
```

Comment:
Using `.get` with `dialog-page` shouldn't be needed, as you're assured for it to exist after mount is executed